src/wombats/components/configuration.clj

```diff
@@ -19,11 +19,12 @@
 (defn- get-config-files
   "Determins the files that should be used for configuration.
```

Comment:
Haha... always gets me... 

@emilyseibert You're not the only one correcting me now 😀